Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS build-env
+WORKDIR /app
+RUN mkdir output
+
+COPY ./ ./
+RUN dotnet restore
+RUN dotnet publish src/Catalyst.Node.POA.CE/Catalyst.Node.POA.CE.csproj -c Debug -o output
+
+# Build runtime image
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2
+RUN apt update -y; apt-get install dnsutils lsof -y
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)